File: app/src/main/java/ntutifm/game/google/net/Parking.java
package ntutifm.game.google.net;

import android.util.Log;

import com.google.android.gms.maps.model.LatLng;

public class Parking {

    private String parkingName;

    private Double parkingLng;

    private Double parkingLat;


    public Parking(String CarParkName, Double PositionLat, Double PositionLon) {
        this.parkingName = "xxxx";
        Log.e("xxxx",PositionLat.toString()+PositionLon.toString());
        this.parkingLng = 12.11;
        this.parkingLat = 12.11;
    }

    public String getParkingName() {
        return "XXXX";
    }

    public LatLng getLatLng() {
        Log.e("xxxx",parkingName);
        return new LatLng(112.1, 112.1);}

}

File: app/src/main/java/ntutifm/game/google/net/apiClass/Parking.kt
package ntutifm.game.google.net.apiClass

import com.google.gson.annotations.SerializedName

data class Parking(
    @SerializedName("CarParkName") val parkingName: String,
    @SerializedName("PositionLat") val latitude: Double,
    @SerializedName("PositionLon") val longitude: Double
)